src/views/PeopleSearch/components/SearchFieldList/index.tsx

```diff
@@ -72,7 +72,7 @@ const relationship_statuses = [
 const searchPageTitle = (
   <>
     Search the
-    <b style={{ color: gordonColors.primary.blue }}> Gordon </b>
+    <b style={{ color: gordonColors.primary.cyan }}> Gordon </b>
```

Comment:
Thanks again for paying attention to consistency issues. However, if this is inconsistent with other pages (which I know it currently is with the involvements page), then it is the other pages that need to change, not this one. 

This text was previously cyan, but we changed it to blue because the cyan on white is insufficient color contrast for accessibility.

Reply:
A lot of pages don't use a header at the top of each page, but on pages with multiple tabs (like Involvements or Events) I thought it made more sense for each tab to have a header so they all looked the same, then I wanted to add headers across the site so it is the same everywhere. Should I leave them in or leave them as is?

For the cyan color, there are other places on the website that use cyan on white (like Involvements or Events), but this is the only page I could find with a dark blue "Gordon" on white. Should I still change it back to dark blue?

Reply:
Yeah, we should definitely leave this one as blue. We should probably also make an issue to find everywhere that has cyan on white because that is not accessible to people with certain disabilities.